to_do_list_part_two.py

```diff
@@ -1,7 +1,16 @@
 def add_task(task_list):
-    add_your_task = input("Enter the task:    ")
-    task_list.append(add_your_task)
-    print(f"{add_your_task} has been added to the list.")
+    task = input("Enter the task: ")
+
+    priority = input("Enter the priority(high, medium, low): ")
+    deadline = input("Enter the dead line(YYYY-MM-DD): ")
```

Comment:
In this line, deadline is str type.
We should discuss about the date type of the deadline: just leave it in str type or convert to datetime type

Reply:
We decided to keep deadline as str type.
no need to change.